contract/test/endo/Makefile

```diff
@@ -0,0 +1,41 @@
+STATE=$(shell endo where state)
+PET=$(STATE)/pet-store
+
+# ag-trade package is currently in finquick
+# https://github.com/dckc/finquick
+AG_TRADE=~/projects/finquick/packages/ag-trade
+
+reserve-add: reserve-add.js $(PET)/alice
+	endo run reserve-add.js 1 IST -p alice
+
+$(PET)/alice: $(PET)/alice-wk $(PET)/fresh-trade-id
+	endo mkguest alice
+
+	endo send alice @wallet:alice-wk
+	endo send alice @fresh:fresh-trade-id
+	endo inbox --as alice
+	# these 0/1 message numbers are fragile
+	endo adopt 0 wallet --as alice
```

Comment:
Might be better(ish) to have `endo send` print a large random number and send it with the message to alice so the id could be transmitted “out of band” and safely used to answer. This is awkward because the numbers are assigned by the receiver.